demo/index.jsx

```diff
@@ -140,7 +140,7 @@ class App extends Component {
 						<Link href="/zustand" activeClassName="active">
 							zustand
 						</Link>
-						<Link href="/redux-toolkit" activeClassName="active">
+						<Link href="/demo/redux-toolkit" activeClassName="active">
```

Comment:
I don't think this should change?

```suggestion
						<Link href="/redux-toolkit" activeClassName="active">
```